vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/SqlResultBase.java

```diff
@@ -18,15 +18,18 @@
 package io.vertx.sqlclient.impl;
 
 import io.vertx.sqlclient.SqlResult;
+import io.vertx.sqlclient.PropertyKind;
 
 import java.util.List;
+import java.util.Map;
 
 public abstract class SqlResultBase<T, R extends SqlResultBase<T, R>> implements SqlResult<T> {
 
   int updated;
   List<String> columnNames;
   int size;
   R next;
+  Map<PropertyKind<?>, Object> sqlResultProperties;
```

Comment:
just call it `properties`

Reply:
updated in https://github.com/eclipse-vertx/vertx-sql-client/pull/392/commits/7135bbeae5cfe7e8a038e23ba45fd3c0a75c2b98.